### DISCUSSION TITLE
Is there a way to expand all message views in thread?

### DISCUSSION THREAD

**Discussion by JohnCido** [InboxSDK/InboxSDK#1230] [Q&A]
We're trying to get the data of all emails in a thread. Gmail itself has a Expand all button in the subject toolbar. Is there a way to trigger it in Inbox so all MessageView could be loaded?

**Comment by Macil** [comment]
I don't think the InboxSDK has a feature for that.

You can select the element itself with `document.querySelector()` and call the `.click()` method on it. (Do note that Gmail keeps a lot of elements in the DOM for pages that aren't currently visible and has many buttons with the same class names, so you may want to use `document.querySelectorAll()` during testing to make sure your selector only selects the elements you want.)
